win32/src/win32file.i

```diff
@@ -419,39 +419,44 @@ PyObject *py_DeviceIoControl(PyObject *self, PyObject *args, PyObject *kwargs)
 
 	if (!PyWinObject_AsHANDLE(obhFile, &hDevice))
 		return NULL;
-	if (!PyWinObject_AsReadBuffer(obInBuffer, &InBuffer, &InBufferSize, TRUE))
+    BufferView out_buf, in_buf(obInBuffer, false, true); // None Ok
```

Comment:
This patch has introduced spaced into a file that previously had only tabs, so tabs and spaces are now mixed here.

Reply:
solved by updated PR